apple/Sources/Sargon/Extensions/Swiftified/TransactionManifest+Swiftified.swift

```diff
@@ -0,0 +1,14 @@
+extension TransactionManifest: @unchecked Sendable {}
+
+extension TransactionManifest: CustomStringConvertible {
+	public var description: String {
+		transactionManifestToString(manifest: self)
+	}
+}
+
+#if DEBUG
+extension TransactionManifest {
+	public static let placeholder: Self = newTransactionManifestPlaceholder()
+	public static let placeholderOther: Self = newTransactionManifestPlaceholderOther()
```

Comment:
Will these follow the `sample()` and `sampleOther()` convention?

Reply:
Just fixed it, find replace all: https://github.com/radixdlt/sargon/pull/14/commits/23e66882c33a4001582cb3dac4286581f15c1f84